### PR TITLE
feat: add verify email card component

### PR DESCRIPTION
**Pull request by bhupeshpr25**
Added a verify email card component.

### CODE REVIEW

File: src/pages/templates/forms/authentication/verifyEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          color={useColorModeValue('gray.800', 'gray.400')}>
+          username@mail.com
+        </Center>
+        <FormControl>
+          <Center>
+            <HStack onChange={handleFocus} autoFocus={true} spacing={4}>
+              <Input maxLength={1} height="10" width="12" />
+              <Input maxLength={1} height="10" width="12" />
+              <Input maxLength={1} height="10" width="12" />
+              <Input maxLength={1} height="10" width="12" />
```
Comment (hauptrolle) [inline src/pages/templates/forms/authentication/verifyEmail.tsx:56]: I think we can use the Pin Input Component here: https://chakra-ui.com/docs/components/pin-input

**Review by hauptrolle — CHANGES_REQUESTED** [review]
Thank you for your contribution, that is a nice template!
I just have one small adjustment I want you to make.

File: src/pages/templates/forms/authentication/verifyEmail.tsx (hunk 1 above)
Comment (bhupeshpr25) [reply src/pages/templates/forms/authentication/verifyEmail.tsx]: thanks for the heads up. You are correct pin input is a better choice.

**Review by hauptrolle — APPROVED** [review]

**Comment by hauptrolle** [comment]
@all-contributors please add @bhupeshpr25  for code

**Pull request merged**